spec/features/brexit_checker/question_results_spec.rb

```diff
@@ -143,4 +156,20 @@ def action_is_shown(key)
       expect(page).to have_link(action.guidance_link_text, href: action.guidance_url)
     end
   end
+
+  def and_the_tourism_link_should_have_tracking_analyitics
+    action = BrexitChecker::Action.find_by_id("T063")
+    expect(page).to have_css(".govuk-link[href='#{action.guidance_url}'][data-track-action='Your business or organisation - 1.2 - Guidance']")
+
+  end
+
+  def and_the_ce_mark_link_should_have_tracking_analyitics
+    action = BrexitChecker::Action.find_by_id("T001")
+    expect(page).to have_css(".govuk-link[href='#{action.guidance_url}'][data-track-action='Your business or organisation - 1.1 - Guidance']")
+  end
+
+  def and_the_pet_link_should_have_tracking_analyitics
+    action = BrexitChecker::Action.find_by_id("S009")
+    expect(page).to have_css(".govuk-link[href='#{action.guidance_url}'][data-track-action='You and your family - Visiting the EU - 1.2 - Guidance']")
+  end
```

Comment:
We could also add some checks for the ecommerce attributes on the links - these should be fairly easy to test as there's no logic in constructing the values, it should just be:

`data-ecommerce-path="[the url]"` and `data-ecommerce-row`